addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorBaseHandler.java

```diff
@@ -66,10 +69,13 @@ void calculateBatteryLevelFromVoltage(Integer voltage) {
         Integer battLevel = (int) ((float) (voltage - VOLTAGE_MIN_MILLIVOLTS)
                 / (float) (VOLTAGE_MAX_MILLIVOLTS - VOLTAGE_MIN_MILLIVOLTS) * 100);
         updateState(CHANNEL_BATTERY_LEVEL, new DecimalType(battLevel));
-        if (battLevel <= BATT_LEVEL_LOW) {
+        int lowThreshold = isBatteryLow ? BATT_LEVEL_LOW + BATT_LEVEL_LOW_HYS : BATT_LEVEL_LOW;
```

Comment:
I was mistaken because I thought you manipulated the actual battery level but you alter the threshold